### PR TITLE
feat: Switch docker compose input to k8s manifest

### PR DESCRIPTION
**Pull request by laurentluce**
Tested in codespace.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Follow these steps to explore the Kardinal Playground.
    - Look for the port labelled "voting-app-prod" and open it in your browser
    - Click on the voting buttons to generate some traffic
   
-   **Note**: Codespaces port forwarding can be flaky. If you immediately click on the toast that pops up when a port is fowarded, it can be too fast and the port tunnel will shut down. If that happens, just run `./scripts/forward.sh` to set up the forwarding again. Then, don't click on the toast - instead, let it run, wait a tick, and open the port in the "ports" tab.
+   **Note**: Codespaces port forwarding can be flaky. If you immediately click on the toast that pops up when a port is fowarded, it can be too fast and the port tunnel will shut down. If that happens, just run `./scripts/forward.sh` to set up the forwarding again. Then, don't click on the toast - instead, let it run, wait 15 seconds, and open the port in the "ports" tab.
```
Comment (laurentluce) [inline README.md:55]: I made this comment a bit more explicit.  I am not a native speaker but I never heard this expression in the US.

Comment (h4ck3rk3y) [reply README.md:55]: Makes sense.

File: voting-app-demo/k8s-manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+      labels:
+        app: voting-app-ui
+        version: v1
+    spec:
+      containers:
+        - name: voting-app-ui
+          image: kurtosistech/demo-voting-app-ui
```
Comment (h4ck3rk3y) [inline voting-app-demo/k8s-manifest.yaml:111]: do we use the image from hub or do we use the images built by startup.sh?

File: voting-app-demo/k8s-manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
+apiVersion: v1
```
Comment (h4ck3rk3y) [inline voting-app-demo/k8s-manifest.yaml:1]: should we drop compose?

**Review by h4ck3rk3y — APPROVED** [review]

**Pull request merged**